frigate/video.py

```diff
@@ -119,27 +126,45 @@ def __init__(self, name, config, prepped_frame_queue, mqtt_client, mqtt_prefix):
         self.config = config
         self.detected_objects = []
         self.recent_frames = {}
-        self.rtsp_url = get_rtsp_url(self.config['rtsp'])
+        if ('rtsp' in config):
+            print('Found rtsp camera config.')
+            self.stream_url = get_rtsp_url(self.config['rtsp'])
+            self.ffmpeg_input_args = self.config.get('ffmpeg_input_args', [
+                '-avoid_negative_ts', 'make_zero', 
+                '-fflags', 'nobuffer',
+                '-flags', 'low_delay',
+                '-strict', 'experimental',
+                '-fflags', '+genpts+discardcorrupt',
+                '-vsync', 'drop',
+                '-rtsp_transport', 'tcp', 
+                '-stimeout', '5000000', 
+                '-use_wallclock_as_timestamps', '1'
+            ])
+        elif ('rtmp' in config):
+            print('Found rtmp camera config.')
+            self.stream_url = get_rtmp_url(self.config['rtmp'])
+            self.ffmpeg_input_args = self.config.get('ffmpeg_input_args', [
+                '-avoid_negative_ts', 'make_zero', 
+                '-fflags', 'nobuffer',
+                '-flags', 'low_delay',
+                '-strict', 'experimental',
+                '-fflags', '+genpts+discardcorrupt',
+                '-vsync', 'drop',
+                '-use_wallclock_as_timestamps', '1'
+            ])
```

Comment:
I figured this would stay as a large object since I believe in the future we may find more differences between the two protocols (and this allows for other protocols down the road).